bin/test-lxd-images

```diff
@@ -221,6 +221,8 @@ for url in $(lxc query "/1.0/instances" | jq -r .[]); do
 
     if [ -z "${address}" ]; then
         echo "FAIL: No network interface: ${name}"
+        # Show the network state
+        lxc query "${url}/state" | jq -r ".network"
```

Comment:
Any reason not to show a simple `lxc info` here? that tends to be slightly more readable :)